content/source/docs/enterprise/private/install-ami.html.md

```diff
@@ -87,19 +91,22 @@ These variables can be populated, but they have defaults that will be used if yo
 * `arn_partition` Used mostly for govcloud installations. Example: `aws-us-gov`
 * `internal_elb` Indicate to AWS that the created ELB is internal only. Example: `true`
 * `startup_script` Shell code that should run on the first boot. This is explained in more detail below.
-* `external_security_group_id` The ID of a custom EC2 Security Group to assign to the ELB for "external" access to the system. By default, a Security Group will be created that allows ingress port 80 and 443 to `0.0.0.0/0`.
-* `internal_security_group_id` The ID of a custom EC2 Security Group to assign to the instance for "internal" access to the system. By default, a Security group will be created that allows ingress port 22 and 8080 from `0.0.0.0/0`.
+* `external_security_group_ids` The IDs of existing EC2 Security Groups to assign to the ELB for "external" access to the system. By default, a Security Group will be created that allows ingress to ports 80 and 443 from `0.0.0.0/0`.
+* `internal_security_group_ids` The IDs of existing EC2 Security Groups to assign to the instance for "internal" access to the system. By default, a Security group will be created that allows ingress to ports 22 and 8080 from `0.0.0.0/0`.
 * `proxy_url` A url (http or https, with port) to proxy all external http/https request from the cluster to. This is explained in more detail below.
+* `no_proxy` Hosts to exclude from proxying, in addition to the default set. (Only applies when `proxy_url` is set.)
```

Comment:
I would replace 'hosts' with 'urls'.

Reply:
The value is literally hosts, however. Not URLs.